jf_agent/agent_logging.py

```diff
@@ -273,16 +397,21 @@ def print_to_log(*args):
     )
 
     logging.basicConfig(
-        level=config.level, datefmt=config.datefmt, handlers=config.handlers,
+        level=config.level,
+        datefmt=config.datefmt,
+        handlers=[logfile_handler, console_log_handler, log_queue_handler],
```

Comment:
Changed this to the list of handlers, there was an issue parsing the handlers from the dataclass attribute